hypha/apply/funds/templates/funds/coapplicant_invite_landing_page.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base-apply.html' %}
+
+{% load i18n %}
+
+{% block user_menu %}
+    <div></div>
+{% endblock user_menu %}
+
+{% block content %}
+    <div class="bg-gray-100 max-w-[70%] mx-auto mt-6 px-10 pt-6 pb-4 border border-gray-300 rounded-sm">
+        {% if is_valid %}
+            <p>{% trans "You've been invited to join the submission" %} "{{ invite.submission.title }}" {% trans "as a co-applicant with email" %} <i>{{ invite.invited_user_email }}</i>. {% trans "Please respond to the invitation by choosing accept or decline." %}</p>
+            <p>{% trans "If you accept, you’ll be automatically signed up or logged in and taken directly to" %} {% if two_factor_required %}{% trans "two factor authentication and then to" %} {% endif %} {% trans "the submission." %}</p>
+            <p>{% trans "We recommend updating your profile after accepting the invite. You’ll also have the option to update your name and email in 'My account' section." %}</p>
```

Comment:
These two sentences can be merged it to one I believe.

Reply:
updated